app/Listeners/QueueMessage.php

```diff
@@ -33,20 +33,19 @@ public function handle(QueueMessageRequest $event)
         $resources = $event->resources;
 
         // Build the email.
-        $email = new Email($resources['message'], $resources['contest'], $resources['competition'], $resources['users']);
-
-        foreach ($email->allMessages as $message) {
-            $content = $message['message'];
+        $manager = app(\Gladiator\Services\Manager::class);
+        $email = new Email($resources, $manager);
```

Comment:
So might be easier than always needing to remember to pass in an instance of the `Manager` class when creating an instance of `Email` to add it via the constructor of `Email` but not as an argument:

```php
// Email class

protected $manager;

public function __construct($resources)
{
  $this->manager = app(\Gladiator\Services\Manager::class);
}
```

If it needs it, now it's always included 😉